pkg/specgenutil/volumes_test.go

```diff
@@ -15,39 +15,39 @@ func Test_validChownFlag(t *testing.T) {
 		{
 			name: "U true",
 			args: args{
-				flag: "U=true",
+				flag: "true",
 			},
 			want:    true,
 			wantErr: false,
 		},
 		{
 			name: "U true case does not matter",
 			args: args{
-				flag: "u=True",
+				flag: "True",
 			},
 			want:    true,
 			wantErr: false,
 		},
 		{
 			name: "U is false",
 			args: args{
-				flag: "U=false",
+				flag: "false",
 			},
 			want:    false,
 			wantErr: false,
 		},
 		{
 			name: "chown should also work",
 			args: args{
-				flag: "chown=true",
+				flag: "true",
 			},
 			want:    true,
 			wantErr: false,
 		},
```

Comment:
...and this is now a duplicate test case, so should be removed.

Reply:
removed